syntax = "proto2";

//import "cisco.proto";

// System uptime
message system_uptime {
    //option (cisco_msg).schema_path = "RootOper.SystemTime.Uptime";

    // Host name
    optional string hostname = 50;
    //[(cisco_field).name="Hostname"]
    // Amount of time in seconds since this system     was last initialized
    optional uint32 uptime = 51;
    //[(cisco_field).name="Uptime"];
}